natgeo/mortar
# [SYTO-732] Update fonts

This is a preliminary PR for this ticket https://jira.natgeo.com/browse/SYTO-732

The current mortar architecture is maintained and fonts are applied to all headings classes (e.g `mt2_h1`, `mt2_h2` `.mt2_article > .children` ) without breaking the site. 

If this turns out to be the accepted approach, the remaining work on this ticket would be 
to update .mt2_ to .mt3_ and add media queries to each of the heading tags. 

Let me know what you think. 

To test:

- git checkout branch
- gulp serve
- Visit: http://localhost:3000/
- Witness font difference or check code.
